addons/api/tests/unit/serializers/fragment-string-test.js

```diff
@@ -0,0 +1,49 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Serializer | fragment string', function (hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
```

Comment:
I think this comment is not valid anymore?

Reply:
This is generated along with the "it exists" test by default.  There are a lot of these comments hanging around.  PR to clean this up is welcome.